system/include/emscripten/emscripten.h

```diff
@@ -271,7 +271,19 @@ emscripten_coroutine emscripten_coroutine_create(em_arg_callback_func func, void
 int emscripten_coroutine_next(emscripten_coroutine);
 void emscripten_yield(void);
 
-
+//  node.js : getenv
+#define getenv(name)\
```

Comment:
because does not work returns to me,

node.js <>
node.js <>

	getenv: function(name)	{
		if(typeof _getenv_ret !== 'undefined')_free( _getenv_ret);
		var envar = JSON.stringify(name);
		var  _getenv_ret  = allocate(intArrayFromString(envar), 'i8', ALLOC_NORMAL);
		return  _getenv_ret ;
	}
	
	 char *s = getenv ( "process.env.APPDATA ");
	 printf ( "\n NODE.JS <%s> " ,s  ) ; 
	 printf ( "\n NODE.JS <%s> " , getenv ( "process.env.HOMEDRIVE" ) ) ;
	 printf ( "\n NODE.JS <%s> " , getenv ( "process.env" ) ) ; 

node.js get confused with string, i think it can't resolve , envar correclty